telegram/ext/handler.py

```diff
@@ -17,6 +17,13 @@
 # You should have received a copy of the GNU Lesser Public License
 # along with this program.  If not, see [http://www.gnu.org/licenses/].
 """This module contains the base class for handlers as used by the Dispatcher."""
+import inspect
+import sys
+
+from telegram.ext import Context
+
+has_follow_wrapped = sys.version_info >= (3, 5)
+has_full_argspec = sys.version_info[0] == 3
```

Comment:
Regardless of the issue of do we want bkwd compat here, please use `CAPITAL_LETTERS` for globals.
Also, since these are internal implementation details, they should be prefixed with `_`.